networkx/generators/expanders.py

```diff
@@ -204,3 +211,250 @@ def paley_graph(p, create_using=None):
             G.add_edge(x, (x + x2) % p)
     G.graph["name"] = f"paley({p})"
     return G
+
+
+def maybe_regular_expander(*, n, d, create_using=None, max_tries=100):
```

Comment:
This makes sense, with the keyword arguments there is an implicit naming convention users should know anyway, so we might as well use positional args since everything is described in the docstring on hover anway